apps.json

```diff
@@ -720,7 +720,7 @@
     "homeassistant": {
         "branch": "master",
         "level": 7,
-        "revision": "c7afa218e651ac259ec6889189cd6a05448caaf3",
+        "revision": "HEAD",
```

Comment:
Reminder, if you use HEAD here, you must code on testing and THEN, once you have fully implemented/debugged/whatever, you can merge your code on master.